ovs/extensions/services/upstart.py

```diff
@@ -91,18 +92,27 @@ def add_service(name, client, params=None, target_name=None, additional_dependen
             client.file_write('/etc/init/{0}.conf'.format(target_name), template_file)
 
     @staticmethod
-    def get_service_status(name, client):
+    def get_service_status(name, client, return_output=False):
```

Comment:
You can find several packages for python2 that have various flavours of a  type decorator.